Create Getting started with List draft (#1925)
DevExpress/devextreme-documentation: concepts/05 UI Components/List/00 Getting Started with List/00 Getting Started with List.md

The **List** displays data from a local or remote data storage and allows users to group, select, search, reorder, and delete items.

> In this tutorial, you will learn how to add the **List** to a page, bind it to data, and configure its core features. As a result, you will build the following UI component:

Reviewer: Whenever we add something new, or create something new, I'd use indefinite article: "add a List". 

The phrase `build a component` associates with what we do at DevExpress. We build components. Our customers use them, and configure them. They can build applications or interfaces. So it's just an issue of word choice. 

```suggestion
In this tutorial, you will learn how to add a **List** to a page, bind it to data, and configure its core features. The following control demonstrates the customization result:
```

> In this tutorial, you will learn how to add a **List** to a page, bind it to data, and configure its core features. The following control demonstrates the customization result:

Reviewer 2: ```suggestion
This tutorial explains how to add a **List** to a page, bind it to data, and configure its core features. The following control demonstrates the result:
```

<div class="simulator-desktop-container" data-view="/Content/Applications/20_2/GettingStartedWith/List/index.html, /Content/Applications/20_2/GettingStartedWith/List/index.js, /Content/Applications/20_2/GettingStartedWith/List/index.css"></div>

> Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-list" target="_blank">getting-started-with-list</a>.

Reviewer: It would be better if you use a human-readable name for the repository title. You can also make the changes in the repository Readme file. 

```suggestion
Refer to the following sections for details on each configuration step. You can also find the full code in the following GitHub repository: <a href="https://github.com/DevExpress-Examples/getting-started-with-list" target="_blank">Getting Started with List</a>.
```

> The **List** displays data from the local or remote store and allows users to group, select, search, reorder or delete items.

Reviewer 3: ```suggestion
The **List** displays data from a local or remote data storage and allows users to group, select, search, reorder, and delete items.
```

> In this tutorial, you will learn how to add the **List** to a page, bind data to it, and configure its core features. As a result, you will build the following UI Component:

Reviewer 3: ```suggestion
In this tutorial, you will learn how to add the **List** to a page, bind it to data, and configure its core features. As a result, you will build the following UI component:
```
